guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -15,9 +15,9 @@ functional_areas:
 ## Introduction {#frontend-dev-guide-javascript-js-coding-docblock-overview}
 
 To add {% glossarytooltip 312b4baf-15f7-4968-944e-c814d53de218 %}JavaScript{% endglossarytooltip %} code inline documentation, follow these guidelines. Some parts of Magento code may not comply with this standard, but we are working to gradually improve this. Following these standard is optional for 3rd-party Magento developers, but will help to create consistent, clean, and easy to read inline documentation.
-This standard are a subset of <a href="https://google.github.io/styleguide/javascriptguide.xml" target="_blank">Google JavaScript Style Guide</a> regulations.
+This standard are a subset of [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml" target="_blank) regulations.
 
-<p>Use <a href="http://www.ietf.org/rfc/rfc2119.txt" target="_blank">RFC 2119</a> to interpret the "must," "must not," "required," "shall," "shall not," "should," "should not," "recommended," "may," and "optional" keywords.</p>
+<p>Use [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt" target="_blank) to interpret the "must," "must not," "required," "shall," "shall not," "should," "should not," "recommended," "may," and "optional" keywords.</p>
```

Comment:
Markdown targets don't work like this. Try `{:target="_blank"}` after the link in parentheses.